src/Configurator.php

```diff
@@ -64,6 +64,23 @@ public function unconfigure(Recipe $recipe)
         }
     }
 
+    public function add(string $name, string $class)
+    {
+        if (false === strpos($name, '/')) {
+            throw new \InvalidArgumentException(sprintf('Flex configurator name "%s" must be prefixed with the vendor name, ex. "foo/custom-configurator".', $name));
+        }
+
+        if (isset($this->configurators[$name])) {
+            throw new \InvalidArgumentException(sprintf('Flex configurator with the name "%s" already exists.', $name));
+        }
+
+        if (AbstractConfigurator::class !== get_parent_class($class)) {
```

Comment:
use `is_subclass_of` instead, to allow deeper inheritance.

Reply:
@stof Fixed.